.env

```diff
@@ -1,7 +1,17 @@
 # Environment variable configs for local testing
-POSTGRES_URI='postgresql://cidcdev:1234@localhost:5432/cidc'
 TEST_POSTGRES_URI='postgresql://cidcdev:1234@localhost:5432/cidctest'
 
+
+# Connect to a local postgres instance
+POSTGRES_URI='postgresql://cidcdev:1234@localhost:5432/cidc'
+
+# Connect to our Cloud SQL Postgres instance.
+# Note: POSTGRES_URI must be commented out for this to work
+# CLOUD_SQL_SOCKET_DIR='<YOUR HOME DIRECTORY>/.cloudsql/'
+# CLOUD_SQL_INSTANCE_NAME='cidc-dfci-staging:us-central1:cidc-postgresql-staging'
+# CLOUD_SQL_DB_USER='cidcuser'
+# CLOUD_SQL_DB_NAME='cidc-staging'
```

Comment:
This config makes it easier to test functions locally against the data in the staging database.